optaplanner-docs/src/main/asciidoc/Chapter-Optimization_algorithms/Chapter-Optimization_algorithms.adoc

```diff
@@ -701,8 +695,8 @@ If the changes of a `CustomPhaseCommand` don't result in a better score, the bes
 
 [NOTE]
 ====
-If the `Solver` or a `Phase` wants to terminate while a `CustomPhaseCommand` is still running, it will wait to terminate until the `CustomPhaseCommand` is done, however long that takes.
-The build-in solver phases don't suffer from this problem.
+If the `Solver` or a `Phase` wants to terminate while a `CustomPhaseCommand` is still running, it will wait to terminate until the `CustomPhaseCommand` is done.
```

Comment:
Can we also say "it will wait to terminate until the `CustomPhaseCommand` is _complete_"? "This may take a significant amount of time."

So: "If the `Solver` or a `Phase` wants to terminate while a `CustomPhaseCommand` is still running, it will wait to terminate until the `CustomPhaseCommand` is complete. This may take a significant amount of time." What do you think?